backend-api/template.yaml

```diff
@@ -32,6 +32,18 @@ Parameters:
     Default: none
 
 Mappings:
+  EnvironmentConfiguration:
+    dev:
+      dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
+    build:
+      dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
+    staging:
+      dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
+    integration:
+      dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
```

Comment:
Just _double checking_ this shouldn't be ProductionVariables?

Reply:
I don't think so, we don't tend to use Int values in the Production Dynatrace environment and this is the same configuration as [mobile-id-check-backend](https://github.com/govuk-one-login/mobile-id-check-backend/blob/main/di-ipv-dca-oauth/template.yaml)